mypy/semanal.py

```diff
@@ -1483,7 +1486,8 @@ def visit_import_all(self, i: ImportAll) -> None:
             self.add_submodules_to_parent_modules(i_id, True)
             for name, node in m.names.items():
                 node = self.normalize_type_alias(node, i)
-                if not name.startswith('_') and node.module_public:
```

Comment:
Note that we don't actually need to access its content, we just need to know that it is there, since in this case all names not in ``__all__`` will have ``module_public = False``.